app/controllers/businessactivities/ExpectedAMLSTurnoverController.scala

```diff
@@ -46,20 +50,21 @@ class ExpectedAMLSTurnoverController @Inject() (val dataCacheConnector: DataCach
             (for {
               businessActivities <- cache.getEntry[BusinessActivities](BusinessActivities.key)
               expectedTurnover <- businessActivities.expectedAMLSTurnover
-            } yield Ok(expected_amls_turnover(Form2[ExpectedAMLSTurnover](expectedTurnover), edit, businessMatching)))
-              .getOrElse (Ok(expected_amls_turnover(EmptyForm, edit, businessMatching)))
-          }) getOrElse Ok(expected_amls_turnover(EmptyForm, edit, None))
+            } yield Ok(expected_amls_turnover(Form2[ExpectedAMLSTurnover](expectedTurnover), edit, businessMatching, businessMatching.alphabeticalBusinessActivitiesLowerCase())))
```

Comment:
Maybe just a little formatting to wrap over multiple lines so as more readable?

Reply:
can do

Reply:
In all honestly I think this is as good as the formatting can get without making it really ugly.
<img width="1792" alt="Screenshot 2020-04-01 at 14 49 44" src="https://user-images.githubusercontent.com/33547722/78144823-08994e00-7428-11ea-85c2-58c95e3d98bf.png">

Reply:
<img width="730" alt="Screenshot 2020-04-01 at 14 58 34" src="https://user-images.githubusercontent.com/40767309/78145865-5498c280-7429-11ea-9dc8-c02b97c90bf9.png">

Reply:
Its not a show stopper. I tend to wrap lines over a certain length. ;-)

Reply:
Guess its personal preference but ive refactored